### DISCUSSION TITLE
Leds under 74HC4067

### DISCUSSION THREAD

**Discussion by antomexita** [tttapa/Control-Surface#728] [Q&A]
hi,
i have a teensy LC connect with CD74HC4067 that drive a LEDs.
This is the code:
`USBMIDI_Interface midi;
CD74HC4067 mux = {  7, {12, 11, 10, 9} };
CD74HC4067 mux1 = {  A9, {12, 11, 10, 9} };
NoteButton button0 {
  14,                       
  {MIDI_Notes::C(4), CHANNEL_10}, 
};

NoteLED led {
  LED_BUILTIN,                 // Pin of built-in LED
  {MIDI_Notes::C(4), CHANNEL_1}, // Note C4 on MIDI channel 1
};

void setup() {
  Control_Surface.begin(); // Initialize Control Surface
  mux1.begin();
 mux.begin();
}

void loop() {
  Control_Surface.loop(); // Update the Control Surface
}` 
Now the Led of teensy respond when my PC send a C4 note, but if i change the LED_BUILTIN with "mux.pin(0)"  the led under the mux not respond. If exist, where find the solucion?  
Many thanks

**Comment by Veroledez** [comment]
Hola, la instancia mux implementada por controlsurface solo es para entrada analogica  o digital, no funciona como salida, para salida debes usar un registro de desplazamiento como el 74hc595, o el max7219 o leds ws2812.
Si aun quieres usar mux para salidas debes usar otra libreria como roxmux, y crear una clase donde utilices dicha libreria, ten en cuenta que un mux no mantiene la salida, asi que para varios leds estos deben estar en un bucle de tal forma que se enciendan por turnos a una buena velocidad para dar la percepcion  de estar encendidos varios a la vez
